packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -69,6 +70,16 @@ class CupertinoButton extends StatefulWidget {
   /// If this is set to null, the button will be disabled.
   final VoidCallback onPressed;
 
+  /// Minimum size of the button.
+  ///
+  /// Defaults to 44.0 which the iOS Human Interface Guideline recommends as the
+  /// minimum tappable area
+  ///
+  /// See also:
+  ///
+  /// * <https://developer.apple.com/ios/human-interface-guidelines/visual-design/layout/>
```

Comment:
I think you need one more space before the `*` to make this work properly on the web site.